packages/docs/content/blog/release-11.0.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Release 11.0
+date: 2024-08-20
+intro: Figma migration, further bundle-size reductions, and replacement of privacy settings dialog
+---
+
+import sketchToFigmaImg from '../../src/images/sketch-to-figma.jpg';
+
+## The Figma migration is complete!
+
+By now most of you have heard about the migration from Sketch to Figma. This move benefits both product teams and our own design system team. For us, we get to leverage [Figma's  multi-mode variable system](https://help.figma.com/hc/en-us/articles/15343816063383-Modes-for-variables) to build and maintain only a single, shared design library and theme it for all CMS brands. That saves us a tremendous amount of time and heartache. For product teams, they'll get our updates faster and with less pain. This isn't a commercial for Figma, but designers just seem to like it better.
+
+<div className="ds-u-text-align--center ds-u-margin-top--3">
+  <img
+    src={sketchToFigmaImg}
+    className="ds-u-display--inline-block ds-u-radius"
+    alt="HealthCare.gov header with the USA banner above it"
+  />
+</div>
+
+Our migration to Figma also prompted us to evolve our own tokens system in order to achieve greater parity and integration between code and design. We developed some internal tools to facilitate bi-directional syncing, enabling our design team to update tokens within Figma and sync them back to our code repository effortlessly. Any synchronization with code happens through version control, which is easy to audit and means engineers can discuss changes on GitHub. It creates a conversation between design and engineering with a better paper trail that is easier to document. Ultimately that's a benefit to product teams.
+
+In the process of syncing with Figma, we took the opportunity to refine many of our design tokens. These changes are a natural part of the evolution of the design system, but it does result in some breaking changes for the CSS variables that are generated from our design tokens. [Details of the CSS-variable changes can be found in our release notes](/REPLACE-ME). We hope that you'll accept a new feature, though, in exchange for making you update a few variables: Most component-level tokens are now defined by referencing theme-level tokens like `--color-primary`, which means you can make high-level customizations and see them trickle down to multiple components. We hope it also makes it easier for developers to see the relationship between tokens across the system without having to store hexadecimal codes in their heads.
```

Comment:
Feels like this should be a new sentence rather than separated by a colon?
"... in exchange for making you update a few variables: Most component-level tokens are now defined by referencing..."

Reply:
I was disregarding modern colon rules here. Perhaps I've been reading too many early 20th century British authors lately, but what I was trying to do was offer up the second sentence by way of the first one in a way that wouldn't lose the link between them, especially because it's just a supporting sentence and not its own paragraph. However, I also see that the sentence that follows it concludes the idea of this sentence and not necessarly of the intro sentence. This paragraph is about more than just this kind of breaking change, I guess. Maybe we need an additional concluding sentence about the high-level benefits of "refining our tokens"? And back to the sentence we're talking about, would a semicolon do?

Reply:
What do you think about this ending? Too much?

> We hope the variable referencing [changed this to be more specific] also makes it easier for developers to see the relationship between tokens across the system without having to store hexadecimal codes in their heads. Overall we're working towards tokens that are better organized and more consistent and understandable, and we thank you for walking with us on this journey.

Reply:
No, this is good. I appreciate the humanizing and poetic touch of the last sentence.